aries_cloudagent/resolver/did_resolver.py

```diff
@@ -42,20 +43,28 @@ async def _resolve(
         profile: Profile,
         did: Union[str, DID],
         service_accept: Optional[Sequence[Text]] = None,
+        *,
+        timeout: Optional[int] = None,
```

Comment:
why not initialize to 30 instead of None? That would make it more evident that the time-out default is 30 seconds.

Reply:
Fair question; using the optional enables expression that the timeout wasn't set. Since we call `_resolve` from two other methods, and we want the timeout to apply to both and we want a default of 30 seconds when the timeout isn't set, we would have to set the default timeout to 30 on both the `resolve` and `resolve_with_metadata` methods or else make sure the timeout wasn't included in the call to `_resolve`. It ends up being cleaner to set the default when timeout is None. But I think there's a middle ground. I'll put a constant on the class that has the default timeout value so it's clearer what it is and how it's used at a glance.